src/GovUk.Education.ExploreEducationStatistics.Admin/Migrations/ContentMigrations/20250812152120_EES6411_AddingUniqueContraintsToRoleInviteTables.cs

```diff
@@ -0,0 +1,120 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Migrations.ContentMigrations
+{
+    /// <inheritdoc />
+    public partial class EES6411_AddingUniqueContraintsToRoleInviteTables : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropIndex(
+                name: "IX_UserReleaseInvites_ReleaseVersionId",
+                table: "UserReleaseInvites");
+
+            migrationBuilder.DropIndex(
+                name: "IX_UserPublicationInvites_PublicationId",
+                table: "UserPublicationInvites");
+
+            migrationBuilder.AlterColumn<string>(
+                name: "Role",
+                table: "UserReleaseInvites",
+                type: "nvarchar(450)",
+                nullable: false,
+                oldClrType: typeof(string),
+                oldType: "nvarchar(max)");
+
+            migrationBuilder.AlterColumn<string>(
+                name: "Email",
+                table: "UserReleaseInvites",
+                type: "nvarchar(450)",
+                nullable: false,
+                oldClrType: typeof(string),
+                oldType: "nvarchar(max)");
```

Comment:
We wouldn't expect email addresses > 450 characters which would break this migration so I think this is fine.

I was just reading up and there is a rule around the max length for email addresses, so in theory we could set a shorter limit, but I don't think we need to do that here.